dns/plugin.spec.yaml

```diff
@@ -1,23 +1,23 @@
 plugin_spec_version: v2
 extension: plugin
 products: [insightconnect]
-name: dig
-title: Dig
-description: Dig is used for forward and reverse DNS lookups
-version: 1.0.7
+name: dns
```

Comment:
We need to keep this `dig` so that users can upgrade (somewhat) seamlessly from the old plugin to the new plugin. The `name` property is only used internally, so it won't be viewable by users. Unfortunately you will also have to change the directory name back to `dig`.